content/en/docs/reference/command-line-tools-reference/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Command line tools reference"
+weight: 60
+toc_hide: true
```

Comment:
Maybe this is why it isn't showing up?

Reply:
OH, it totally is--camel case, not snake case. Thanks!